mmv1/third_party/terraform/services/oracledatabase/data_source_oracle_database_cloud_exadata_infrastructures.go

```diff
@@ -0,0 +1,109 @@
+package oracledatabase
+
+import (
+	"fmt"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-provider-google/google/tpgresource"
+	transport_tpg "github.com/hashicorp/terraform-provider-google/google/transport"
+)
+
+func DataSourceOracleDatabaseCloudExadataInfrastructures() *schema.Resource {
+	dsSchema := map[string]*schema.Schema{
+		"project": {
+			Type:        schema.TypeString,
+			Optional:    true,
+			Description: "The ID of the project in which the dataset is located. If it is not provided, the provider project is used.",
+		},
+		"location": {
+			Type:        schema.TypeString,
+			Required:    true,
+			Description: "location",
+		},
+		"cloud_exadata_infrastructures": {
```

Comment:
Could we possibly add this field to the test?

Reply:
I have added to check if not null values are set, We can not add to check for all cloud_exadata_infrastructure instances because it might fail considering the list operation may list any resource that is still getting provisioned by other resource (cloud_exadata_infrastructures ) test